### DISCUSSION TITLE
Tumblr Batch Block

### DISCUSSION THREAD

**Discussion by kitblog** [AprilSylph/XKit-Rewritten#358] [Suggestion Box]
Would it be possible to implement something like the "Tumblr Batch Block" extension into XKit Rewritten?

https://github.com/realityfabric/Tumblr-Batch-Block

The ability to import/export blocklists and block large amounts of users was useful for harassment protection. Now that TBB is no longer being actively developed since the dev left Tumblr/is looking to hand it off, I was wondering if it could be brought back somehow as there's no other working alternative extensions.

**Comment by marcustyphoon** [comment]
I cannot recall whether this has been discussed before - probably not; I am likely thinking of editing filtered tags - but along with this discussion, do submit this as a feature request at https://www.tumblr.com/support! I don't know if Staff would ultimately consider it a good feature for the native site, but I think they would at least consider it if enough users showed they'd use it.

**Comment by Dylawa** [comment]
I apologize, I'm not sure what the proper way is to comment in this old thread, and I'm sure this counts as "necro-ing" the discussion, but seeing as it is now halfway through 2023 and Tumblr still has not implemented this feature, I think it'd be safe for me to be the one to ask for this feature again? I sent feedback to Tumblr Support as well, as Marcustyphoon suggested.

I checked the Issues page, checked the projects page, and scrolled through the Suggestion box without additional filters to make sure this wasn't currently being developed before replying, but I'm so sorry if I missed something anyways. It's probably more important now than ever to be able to block multiple accounts at once/quickly and share those block lists (even if it's just with our own personal side blogs). I feel this way with the surging return of bots, as well as new users not understanding Tumblr (or really internet) etiquette and openly and proudly harassing other and older users.

**Comment by marcustyphoon** [comment]
In general, I don't think we plan to do this. Mass-blocking bot users doesn't really serve any purpose besides slightly slowing down the site once they are banned, and a vaguely similar thing is true of harassment: instances of inappropriate behavior should be reported so they can be removed for everyone.

Copying your blocklists between your sideblogs sounds like a useful utility, though.

**Comment by SpeedyElephant** [comment]
I hope it's ok to bump this. I would _really_ appreciate being able to copy my blocklist between sideblogs. It doesn't have to be a share-able blocklist.

**Comment by CharaViolet** [comment]
I was actually about to ask for a feature to make blocking apply to all sideblogs too, glad I checked this post since it seems to be going in a similar direction
